### ISSUE TITLE
nothing working in just home app

### ISSUE THREAD

**Issue by roberkane212** [tuya/tuya-homebridge#320]
I have been using the pluggin jusr fine, however after homebridge update none of my devices are working in homekit. they dont give a no responce error just does nothing when push in app. all things good on tuya site, and all things working from smartlife app

**Comment by damanveen** [comment]
same here on my end. Hasn't worked in a while

**Comment by Pitxoio** [comment]
This has also happened to me. Upon checking the logs from Homebridge, it was because my subscription to iOT core on the tuya platform website expired. 
I think Tuya must have changed something on their free tier subscription that doesn't allow Homebridge to communicate back information from the plugin to Tuya.

**Comment by tablatronix** [comment]
Same just stopped working one day, accessory is there, toggling it does nothing, no logs from this plugin are even generated when using the accesory.

**Comment by marcham93** [comment]
Any updates here on whether this is the plugin or Tuya? As reported above, toggling the switch does nothing, but Tuya IoT does show something received from the plugin via the Cloud API.

**Comment by DavKirsch** [comment]
I had this same issue and just solved it today. About a month ago, all my devices on the Tuya Platform bridge stopped working in Homekit. They were still working on the Smart Life app though...

After a lot of searching, I discovered the IoT Core Services on the Tuya IoT Developer Platform had been working via a free trial and that had expired. There was an orange button to request an extension of the trial (as buying the bundle was $25K+). I asked for an extension for 6 months for my Hoobs Smart Hub, which was granted by the next morning. Then, I restarted the Bridge in HOOBs and all my devices were cached again and working in Homekit!! Took me a month to solve but everything is up and running. 

FYI, You will need to repeat this process every 6 months though, the only downside.

[Workflow with Tuya.pdf](https://github.com/tuya/tuya-homebridge/files/10296124/Workflow.with.Tuya.pdf)
